src/main/java/com/quantumsoft/qupathcloud/converter/qpdata/DataToDcmConverter.java

```diff
@@ -17,78 +17,100 @@
 
 import com.quantumsoft.qupathcloud.dao.Modality;
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
-import org.apache.commons.io.FilenameUtils;
+import java.io.BufferedInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.ZonedDateTime;
+import java.util.Date;
 import org.dcm4che3.data.Attributes;
 import org.dcm4che3.data.Tag;
 import org.dcm4che3.data.UID;
 import org.dcm4che3.data.VR;
 import org.dcm4che3.io.DicomOutputStream;
 import org.dcm4che3.util.StreamUtils;
 import org.dcm4che3.util.UIDUtils;
-import java.io.BufferedInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.time.ZonedDateTime;
-import java.util.Date;
 
+/**
+ * Qpdata to DICOM converter.
```

Comment:
I would add some details on what QpData is..

Reply:
Done